src/constants/indicators.js

```diff
@@ -31,18 +31,18 @@ export default {
   },
   indigenousPopulation: {
     name: 'Indigenous population',
-    shortDescription: 'Indigenous population',
-    longDescription: 'TODO',
+    shortDescription: 'Indigenous status, percentage of total population',
+    longDescription: 'Indigenous status, percentage of total population',
     dataType: DATA_TYPES.NUMBER,
     aggregationMethod: AGGREGATION_METHODS.MEDIAN,
     format: '0[.]0%',
     contextual: true,
     source: 'Indigenous status, % of population',
   },
   population10yr: {
-    name: 'Population 10 year growth rate',
-    shortDescription: 'Growth rate',
-    longDescription: 'TODO',
+    name: '10-year population growth rate',
```

Comment:
We should raise that this is ambiguous. It should highlight that it's the average annual growth, not the growth over 10 years.